src/ResourceManager/Billing/Commands.Billing.Test/SessionRecords/Microsoft.Azure.Commands.Billing.Test.ScenarioTests.BillingPeriodsTests/TestGetBillingPeriodWithName.json

```diff
@@ -1,8 +1,8 @@
 {
   "Entries": [
     {
-      "RequestUri": "/subscriptions/0347d63a-421a-43a7-836a-5f389c79cd07/providers/Microsoft.Billing/billingPeriods/201705-1?api-version=2017-04-24-preview",
```

Comment:
Did you change this manually, or did you re-record?

Reply:
I updated the recordings. I'm investigating why recording isn't working for me, and will update the recordings once I have it working.

Reply:
Re-recorded everything.